### ISSUE TITLE
Can't get Helm version for Helm v3.3.2 and v3.3.3 (was: --name option not recognized)

### ISSUE THREAD

**Issue by vcajes** [tilt-dev/tilt#3788]
when we try to use a tiltfile like

`k8s_yaml(helm('./project/charts/p1', name='project1', namespace='local-project', values=['./values_local.yaml']))`

Error we get:

`--name option not recognized`

Using tilt v0.17.5 and helm v3.3.2

**Comment by nicks** [comment]
Hmmmm...I'm having trouble reproducing this.

1) Can you paste the output of `helm version --client --short`?
2) Does the helm demo work for you? https://github.com/tilt-dev/tilt-helm-demo

**Comment by jburks725** [comment]
Seeing the same on macOS. Here's my Tiltfile fragment:

```
k8s_yaml(helm(
  './chart',
  # The release name, equivalent to helm --name
  name='my-ui',
  # The namespace to install in, equivalent to helm --namespace
  namespace='dev-namespace',
  # The values file to substitute into the chart.
  values=['./scripts/local-values.yaml'],
  # Values to set from the command-line
  set=['tls.enabled=false', 'identity.issuerOverride=http://localhost:8000']
))
```

The generated command that `tilt up` tries to run:
```
❯ helm template /Users/jason/development/my-ui/chart --name my-ui --namespace dev-namespace --values ./scripts/local-values.yaml --set tls.enabled=false --set identity.issuerOverride=http://localhost:30000
Error: unknown flag: --name
❯ tilt version
v0.17.5, built 2020-09-16
❯ helm version --client --short
v3.3.3+g55e3ca0
```

**Comment by nicks** [comment]
if you downgrade to Helm v3.3.1, does that fix the problem?

i can reproduce this problem if i do

```
chmod a+r ~/.kube/config
```

Helm 3.3.2 added a bunch of new warnings that check for things like this, and i wonder if that's interfering with version parsing. (Tilt uses `helm --name` if it can't detect what version of helm you're using)

**Comment by nicks** [comment]
(i think a good short-term fix would be for tilt to error out completely if it can't determine the helm version, which would make this problem less inscrutable)

**Comment by nicks** [comment]
filed as https://github.com/helm/helm/issues/8786

**Comment by jburks725** [comment]
Confirmed that fixing `~/.kube/config` permissions so that `helm` displays no warnings *does* resolve this error.

**Comment by maiamcc** [comment]
Fixed by #3801 (note that the underlying Helm issue was fixed in v3.3.4, see https://github.com/helm/helm/issues/8776
